### ISSUE TITLE
Minecraft error with playerlist

### ISSUE THREAD

**Issue by nitplayz** [gamedig/node-gamedig#304]
So here i have an issue with the playerlist

in the image 
In 1st embed there are 19 players so its just giving me some commas instead of player names
In 2nd embed there are only 12 players so its giving me the correct output with 12 player names
![screenshot](https://user-images.githubusercontent.com/83857989/202221168-14ced88d-c7df-442b-8761-001e4c9b2971.png)

`createEmbed` is the function which creates the embed

```js
Gamedig.query({
      type: 'minecraft',
      host: ip,
      port: port
  }).then((state) => {
    console.log(state)

    let name = state.name
    let map = state.map
    let max = state.maxplayers
    let playerslength = state.players?.length || '0'
    // let players = state.players?.map(p => `>>> \`\`\`fix
    // ${p.name}\`\`\``).join('\n') || 'No Players'
    let players = state.players?.map(p => `${p.name}`) || 'No Players'
    let ping = state.ping
    let connect = state.connect;

    console.log('[Map]'.green.bold + map)

    createEmbed(players,playerslength, max, name, ping, connect, interaction.channel.id)
    interaction.reply(`✅`)
 }).catch(e => {
  return console.log(`${e}`.red);
 })
```

Function for embed:

```js
function createEmbed(players, onlineplayers, totalplayers, name, ping, connect, channel){
    const embed = new EmbedBuilder()
      .setTitle(`\`${mc.servername}\` Status: 🟢 Online`)
      .setFooter({text: `Last Updated:`, iconURL: mc.serverIcon})
      .setTimestamp()
      .setDescription(`**Playerlist:** \n${players} `)
      .addFields(
          {name: `**Players Online:**`, value:`\`${onlineplayers}/${totalplayers}\``, inline: true},
          {name: `**MOTD:**`, value: `\`${name}\``, inline: false},
          {name: `🔃 **Ping:**`, value: `\`${ping}\``, inline: true},
          {name: `**IP:**`, value: `\`${connect}\``, inline: true},
          {name: `**Version:**`, value: `${mc.serverVersion}`, inline:true } 
      )

      interaction.client.channels.cache.get(channel).send({embeds: [embed]})
    }
```

Please help me with this error i have given my best and every possible try

**Comment by CosminPerRam** [comment]
Many Minecraft servers do not respond with players data, this could be added as documentation.

**Comment by podrivo** [comment]
I'll make sure to add this info in our docs! (:

**Comment by CosminPerRam** [comment]
Closed by #492.
Feel free to reopen if something comes up (:
